docs/api/rundeck-api.md

```diff
@@ -1069,24 +1069,318 @@ Response:
+
+---
+    401 Unathorized
+
+API Key is associated with a user or role with insufficent permissions to perform the action.
+
+Headers  
+`Content-Type: application/json`
+
+Body
+```json
+{
+  "msg":"Unauthorized"
+}
+```
+
+### Delete a Single Config ####
+    DELETE /api/36/config/delete
+
+Delete a single config by key and strata.
+
+**Request:**
+
+Headers
+* `Content-Type` Required
+  * Accepts: `application/json`
+
+Body
+* `key` Required
+  * Represents the config to be deleted.
+  * Accepts: Any `string`
+* `strata` Optional
+  * Whether the config should apply to the current server (`Server`), or to all servers in the cluster (`default`).
+  * Accepts: `Server`, `default`
+  * Default: `default`
+
+Example JSON
+```json
+  {
+    "key": "myCustomConfig",
+    "strata": "default"
+  }
+```
+
+**Response**
+
+    204 No Content
+
+The config was deleted successfully.
+
+---
+    401 Unathorized
+
+API Key is associated with a user or role with insufficent permissions to perform the action.
+
+Headers  
+`Content-Type: application/json`
+
+Body
+```json
+{
+  "msg":"Unauthorized"
+}
+```
+
+---
+    404 Not Found
+
+A config with the input config key was not found.
+
+Headers  
+`Content-Type: application/json`
+
+Body
+```json
+{
+  "error":true,
+  "apiversion":40,
+  "errorCode":"rundeckpro.config.api.error.not.found",
+  "message":"System Configuration Not Found"
+}
+```
+
+### List All Current Configurations ####
+    GET api/36/config/list
+List all existing configs and their properties.
 
 **Request:**
 
+Headers
+* `Content-Type` Optional
+  * Accepts: `application/json`
+
+**Responses:**
+
+    200 OK
+
+Call was successful. A list of all configs is returned.
+
+Headers  
+`Content-Type: application/json`
+
+Body
+```json
+{
+  "Plugins": {
+    "AWS": [
+      {
+      "key": "rundeck.nodeService.nodeCache.enabled",
+      "strata": "default",
+      "label": "Node Cache",
+      "value": true
+    }
+    ]
+  },
+  "Custom": {
+    "": [
+      {
+        "key": "myCustomConfig",
+        "value": "myConfigValue",
+        "strata": "Server",
+        "category": "Custom",
+        "visibility": "Custom"
+      },
+      {
+        "key": "myCustomConfig2",
+        "value": "myConfigValue2",
+        "strata": "default",
+        "category": "Custom",
+        "visibility": "Custom"
+      }
+    ]
+  }
+}
+```
+
+---
+    401 Unathorized
+
+API Key is associated with a user or role with insufficent permissions to perform the action.
+
+Headers
+
+`Content-Type: application/json`
+
+Body
+```json
+{
+  "msg":"Unauthorized"
+}
+```
+
+### Refresh Configurations from Properties File ####
     POST /api/36/config/refresh
-    
+
+Make the Rundeck server re-read the config properties file.
+
+**Request:**
+
+Headers
+* `Content-Type` Optional
+  * Accepts: `application/json`
+
 **Response**
 
+    200 OK
+
+The call was successful. Rundeck will reread the config properties file.
```

Comment:
I think it's important to note here that some configs require a restart of Rundeck and this refresh will only work on certain configs.

Reply:
Added as a note